app/scripts/router.js

```diff
@@ -207,6 +209,24 @@ function (
       return new View(viewOptions);
     },
 
+    _checkForRefresh: function () {
+      var storage = Storage.factory('sessionStorage', this._window);
+      var refreshMetrics = storage.get('last_page_loaded');
+      var currentView = this.currentView;
+      var screenName = currentView.getScreenName();
+
+      if (refreshMetrics && refreshMetrics.view === screenName && this.metrics) {
+        currentView.logScreenEvent('refresh');
```

Comment:
Can you add this event to [client-metrics.md](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md) under [generic-events](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md#generic-events)?